<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1226</width>
    <height>804</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Анализатор культи</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/resources.qrc">
    <normaloff>:/icons/app_icon.svg</normaloff>:/icons/app_icon.svg</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextUnderIcon</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1">
      <property name="spacing">
       <number>6</number>
      </property>
      <item>
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="midLineWidth">
         <number>0</number>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="ModelWidget" name="openGLWidget"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1226</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>28</width>
     <height>28</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionFind"/>
   <addaction name="separator"/>
   <addaction name="actionConnect"/>
   <addaction name="actionAnalyse"/>
   <addaction name="separator"/>
   <addaction name="actionSettings"/>
   <addaction name="actionAbout"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="minimumSize">
    <size>
     <width>282</width>
     <height>242</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>История исследований</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QFrame" name="frame_2">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTreeWidget" name="treeWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <column>
           <property name="text">
            <string>Иследование</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Дата</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_2">
   <property name="minimumSize">
    <size>
     <width>87</width>
     <height>40</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Данные пациента</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="PatientWidget" name="dockWidgetContents_2"/>
  </widget>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/electronics.svg</normaloff>:/icons/electronics.svg</iconset>
   </property>
   <property name="text">
    <string>Соединение</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/good_decision.svg</normaloff>:/icons/good_decision.svg</iconset>
   </property>
   <property name="text">
    <string>Новый пациент</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/decision.svg</normaloff>:/icons/decision.svg</iconset>
   </property>
   <property name="text">
    <string>Найти пациента</string>
   </property>
  </action>
  <action name="actionAnalyse">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/collect.svg</normaloff>:/icons/collect.svg</iconset>
   </property>
   <property name="text">
    <string>Начать исследование</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/settings.svg</normaloff>:/icons/settings.svg</iconset>
   </property>
   <property name="text">
    <string>Настройки</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icons/info.svg</normaloff>:/icons/info.svg</iconset>
   </property>
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ModelWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>modelwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PatientWidget</class>
   <extends>QWidget</extends>
   <header>patientwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
